Mysql Inventory Bridge/src/main/java/net/craftersland/bridge/inventory/wallethook/WalletHandler.java

```diff
@@ -0,0 +1,68 @@
+package net.craftersland.bridge.inventory.wallethook;
+
+import io.mymetaverse.livewallet.api.MetaWalletAPI;
+import io.mymetaverse.livewallet.data.tokens.RegisteredToken;
+import io.mymetaverse.livewallet.utils.TabType;
+import net.craftersland.bridge.inventory.objects.BlackListedItem;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.bukkit.plugin.Plugin;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class WalletHandler {
+
+    private static WalletHandler instance;
+
+    private final Plugin walletPlugin;
+
+    private WalletHandler(Plugin walletPlugin) {
+        this.walletPlugin = walletPlugin;
+    }
+
+    public Set<BlackListedItem> getBlacklistItems() {
+
+        if (!this.getStatus()) return null;
+
+        Set<RegisteredToken> registeredWeapons = MetaWalletAPI.getInstance().getSpecificTokens(TabType.EQUIPMENT);
+        Set<BlackListedItem> blackListedItems = new HashSet<>();
+
+        for (RegisteredToken registeredWeapon : registeredWeapons) {
+
+            if (registeredWeapon.getMaterial() == null) continue;
+
+            if (registeredWeapon.getItemModelData() == null) continue;
+
+
+            if (!registeredWeapon.getItemModelData().matches("[0-9.]+")) continue;
```

Comment:
I believe ModelData cannot be a double, Not sure if I am mistaken sorry otherwise let's change it to an integer to save memory space.

Reply:
Model data is defined as a double on MMOItems, that's why xD

Reply:
Ah. That plugin lol.